run_dir/design/base.html

```diff
@@ -51,30 +51,34 @@
             <ul class="nav navbar-nav navbar-right">
                 <li class="dropdown" id="production">
                     <a class="dropdown-toggle" data-toggle="dropdown" href="#production">Production <span class="caret"></span></a>
-                    <ul class="dropdown-menu" role="menu">
-                        <li><a href="/applications">Application Counts</a></li>
-                        <li><a href="/production/cronjobs">Cronjobs List</a></li>
-                        <li><a href="/nas_quotas">NAS Quotas</a></li>
-                        <li><a href="/flowcells_plot">Flowcells Trends</a></li>
-                        <li><a href="/flowcell_count_plot">Flowcells Counts</a></li>
-                        <li><a href="/reads_total/">Read Count Totals</a></li>
-                        <li><a href="/proj_meta">Compare Project Meta</a></li>
-                        <li><a href="/instrument_logs">View Instrument Logs</a></li>
-                        <li><a href="/workset_placement">Workset Setup</a></li>
-                        <li><a href="http://lims-dashboard.scilifelab.se">Script Dashboard</a></li>
+                    <ul class="dropdown-menu" role="menu"><li class="dropdown-header">General stats</li>
+                    <li><a href="/applications">Application Counts</a></li>
+                    <li><a href="/production/cronjobs">Cronjobs List</a></li><li role="separator" class="divider"></li>
+                    <li class="dropdown-header">Flowcells</li>
+                    <li><a href="/flowcells_plot">Flowcells Trends</a></li>
+                    <li><a href="/flowcell_count_plot">Flowcells Numbers</a></li><li role="separator" class="divider"></li>
+                    <li class="dropdown-header">Tools</li>
+                    <li><a href="/nas_quotas">NAS Quotas</a></li>
+                    <li><a href="/reads_total/">Read Count Totals</a></li>
+                    <li><a href="/proj_meta">Compare Project Meta</a></li>
+                    <li><a href="/instrument_logs">View Instrument Logs</a></li>
+                    </li><li role="separator" class="divider"></li>
```

Comment:
Looks like an extra `</li>` crept in here..

I try to keep HTML lists `<li>`s indented from the `<ul>`, and with each `<li>` element on its own line (eg. including the dividers). Makes it easier to read..